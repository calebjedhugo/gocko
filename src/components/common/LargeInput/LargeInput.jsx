import React, { forwardRef } from "react";
import TextField from "@material-ui/core/TextField";
import styled from "styled-components";

const STextField = styled(TextField)`
    & input {
        font-size: 100px;
        max-width: 200px;
    }
`;

const LargeInput = forwardRef(({ ...rest }, ref) => (
    <STextField
        inputRef={ref}
        autoComplete="off"
        variant="outlined"
        {...rest}
    ></STextField>
));

export default LargeInput;
