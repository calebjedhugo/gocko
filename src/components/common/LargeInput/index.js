import LargeInput from "../LargeInput";

export { default } from "./LargeInput";
