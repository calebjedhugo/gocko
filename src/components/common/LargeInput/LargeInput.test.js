import { render } from "@testing-library/react";
import LargeInput from "./LargeInput";

describe("<LargeInput>", () => {
    it("Matches previous snapshot", () =>
        expect(render(<LargeInput />).baseElement).toMatchSnapshot());
});
