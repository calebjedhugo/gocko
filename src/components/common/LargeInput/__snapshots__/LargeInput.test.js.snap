// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LargeInput> Matches previous snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiFormControl-root MuiTextField-root sc-bdfBwQ gceYuX"
    >
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
      >
        <input
          aria-invalid="false"
          autocomplete="off"
          class="MuiInputBase-input MuiOutlinedInput-input"
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
          style="padding-left: 8px;"
        >
          <legend
            class="PrivateNotchedOutline-legend-2"
            style="width: 0.01px;"
          >
            <span>
              â€‹
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
  </div>
</body>
`;
